File: go/src/handler.go
package main

import (
	// mesh "github.com/digitalcompanion-keti/dcf-watcher/go/mesh"
	sdk "github.com/digitalcompanion-keti/dcf-watcher/go/pb"
)

func Handler(req sdk.Request) string {

	// mesh call
	// When using, uncomment the mesh that is commented out on the import.
	//
	// functionName := "<FUNCTIONNAME>"
	// input := string(req.Input)
	// result := mesh.MeshCall(functionName, []byte(input))
	// return result

	// single call
	return string(req.Input)
}

File: python/src/handler.py
import mesh

class Handler:
    def __init__(self):
        pass

    def __call__(self, req):

        # mesh call
        #
        # functionName = "<FUNCTIONNAME>"
        # input = req.input
        # result = mesh.mesh_call(functionName, input)
        # return result

        # single call
        return req.input
